New documentation section for certificates information indicators (#407)
kiali/kiali.io: content/documentation/staging/installation-guide/index.adoc

This allows Kiali to read service mesh resources found in the cluster, but it does not allow Kiali to add, modify, or delete them.

==== Secrets Permissions

Kiali and its operator have reduced permissions for handling secrets due to security reasons.

For some features like link:/documentation/staging/features/#_certificates_information_indicators[Certificates Information Indicators], read permissions for specific secrets in the control plane namespace might be required.

As the following configuration in Kiali CR indicates, the feature will require read permissions for two secrets:

[source,yaml]
----
spec:
  kiali_feature_flags:
    certificates_information_indicators:
      enabled: true
      secrets:
      - dns.example1-service-account
      - dns.example2-service-account
----

These permissions can be configured in two different ways depending on how you install the operator.

When installing the operator using the Helm chart, set the property `secretReader`. Secrets listed in this property will be allowed to be read by the operator and subsequently by Kiali.

By default the secrets listed are:

[source,yaml]
----
secretReader: ['cacerts', 'istio-ca-secret']
----

> When installing the operator with OLM there is no way to update the operator permissions after installing it, the solution is to manually update the cluster role used by the operator as follows:

Reviewer: This is contradictory - first it says "there is no way to update the operator permissions after installing it" and then the following instructions tell you how to update the operator permissions after you installed it. :-)

I think we want this to say:

```suggestion
When installing the operator with OLM there is no way to pre-configure the operator permissions prior to installing the operator. The solution is to manually update the cluster role used by the operator as follows:
```
